src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Transform.cs

```diff
@@ -204,7 +204,7 @@ public Hashtable PropagatedNamespaces
                 CanonicalXmlNodeList namespaces = null;
                 if (reference != null)
                     namespaces = reference._namespaces;
-                else if (signedXml._context != null)
+                else if (signedXml?._context != null)
```

Comment:
@krwq since this is a bug in Desktop, here is the process we should follow
- Port the code as-is (don't introduce a change) -- this would bring the code to par with desktop
- File a new issue for the change you are making
- Tag the issue with `netfx-port-consider` this will flag it for consideration for Desktop
- Follow the Desktop Shiproom process to get this change approved

If we start making fixes as we are porting the code there is a high chance that we are going to forget to port that change back to Desktop.